### ISSUE TITLE
bug/mobile navigation announced by NVDA as a "dialog"

### ISSUE THREAD

**Issue by CorinaMurg** [CorinaMurg/BugReport-WebForEveryone#3]
Currently using a React modal component. The NVDA is announcing as "dialog". Also, the "expanded" state is not announced.

Fix: build the navigation from scratch.
